api/sessiontokenloginprovider_test.go

```diff
@@ -122,3 +122,31 @@ func (s *sessionTokenLoginProviderProviderSuite) Test(c *gc.C) {
 	c.Assert(obtainedSessionToken, gc.Equals, sessionToken)
 	defer func() { _ = apiState.Close() }()
 }
+
+func (s *sessionTokenLoginProviderProviderSuite) TestInvalidSessionTokenLogin(c *gc.C) {
```

Comment:
This should probably be called "TestUnauthorisedLogin" right?
Since that's the error we are expecting in the case under test.

Reply:
Possibly, but also not really. More accurately, it's testing that the error is propagated back to the caller, provided that the error is not a `CodeSessionTokenInvalid` error.